spooky-oysters/bangazon-web-app
# Kb validate product category

## Link to Ticket
Closes #10 

## Description of Proposed Changes

- When user forgets to select product category, they receive a message to select. 
- When user successfully fills out add product form, they are then redirected to the product they created

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout kb-validate-product-category
```

1. Run build
1. Log in, if not already
1. Click "sell a product"
1. Fill out everything except "product category" and submit. You should get an error message product category
1. Select a product category and submit
1. You should be redirected to the product you just created


## Impacted Areas in Application

List general components of the application that this PR will affect:

* ProductCreateViewModel.cs
* ProductController.cs
* Create.cshtml
* Product.cs

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
